### PR TITLE
docs: panel refactor initial build out

### PR DESCRIPTION
**Pull request by chri2547**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: docs/sources/panels-refactor/_index.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
++++
+title = "Panels refactored"
```
Comment (osg-grafana) [inline docs/sources/panels-refactor/_index.md:2]: Why is there a subdir called `panels-refactor`?

File: docs/sources/panels-refactor/add-panels-dynamically/_index.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
++++
+title = "Enable template variables to add panels dynamically"
```
Comment (osg-grafana) [inline docs/sources/panels-refactor/add-panels-dynamically/_index.md:2]:
```suggestion
title = "Add panels dynamically"
```

File: docs/sources/panels-refactor/add-panels-dynamically/_index.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+aliases = ["/docs/grafana/latest/features/panels/panels/"]
+weight = 200
++++
+
+# Enable template variables to add panels dynamically
```
Comment (osg-grafana) [inline docs/sources/panels-refactor/add-panels-dynamically/_index.md:7]:
```suggestion
# Add panels dynamically
```

File: docs/sources/panels-refactor/add-panels-dynamically/_index.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+
+Harness the power of Grafana by configuring the system to automatically and dynamically add panels based on criteria that you define.
```
Comment (osg-grafana) [inline docs/sources/panels-refactor/add-panels-dynamically/_index.md:9]:
```suggestion
Harness the power of Grafana by enabling template variables to automatically and dynamically add panels based on criteria that you define.
```

File: docs/sources/panels-refactor/add-panels-dynamically/_index.md [hunk 5]
```diff
@@ -0,0 +1,13 @@
++++
+title = "Enable template variables to add panels dynamically"
+aliases = ["/docs/grafana/latest/features/panels/panels/"]
```
Comment (osg-grafana) [inline docs/sources/panels-refactor/add-panels-dynamically/_index.md:3]: Is the duplicated `panels` subdirectory correct?

**Review by osg-grafana — CHANGES_REQUESTED** [review]
This is still in draft state, so I am unsure if you would like feedback yet.

**Comment by mjseaman** [comment]
Feedback needed from @pkolyvas and me:
- The first (summary of "why") sentence of a given topic: is it helpful?
- The Before you Begin section: is the list of pre-reqs complete?
- What is missing?
- Is the new nav structure correct? Do these user goals make sense?

Don't worry about:
- The order of the topics.

File: docs/sources/panels-refactor/_index.md (hunk 1 above)
Comment (chri2547) [reply docs/sources/panels-refactor/_index.md]: updated this, resolved.

File: docs/sources/panels/working-with-panels/download-log-results.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
++++
+title = "Download log results"
```
[the remark this answers is not in the document]
Comment (chri2547) [reply docs/sources/panels/working-with-panels/download-log-results.md:2]: Ok, let's sync up about what to title this task.

File: docs/sources/panels/_index.md [hunk 7]
```diff
@@ -1,26 +1,16 @@
...
+description = "Panels"
 aliases = ["/docs/grafana/latest/features/panels/panels/"]
-weight = 70
+weight = 72
 +++
 
-# Panel overview
+# About Grafana panels
 
 The _panel_ is the basic visualization building block in Grafana. Each panel has a query editor specific to the data source selected in the panel. The query editor allows you to extract the perfect visualization to display on the panel.
```
[the remark this answers is not in the document]
Comment (chri2547) [reply docs/sources/panels/_index.md:10]: Good idea, added a link to the data sources content.

File: docs/sources/panels/_index.md [hunk 8]
```diff
@@ -1,26 +1,16 @@
...
-
-## Tips and shortcuts
-
-- Click the graph title and in the dropdown menu quickly duplicate the panel.
-- Click the colored icon in the legend to change a series color or the y-axis.
-- Click series name in the legend to hide series.
-- Ctrl/Shift/Meta + click legend name to hide other series.
-- Hover your cursor over a panel and press `e` to open the panel editor.
-- Hover your cursor over a panel and press `v` to open the panel in full screen view.
+{{< section >}}
```
[the remark this answers is not in the document]
Comment (chri2547) [reply docs/sources/panels/_index.md:16]: Changed the order accordingly.

File: docs/sources/panels/working-with-panels/_index.md [hunk 9]
```diff
@@ -0,0 +1,15 @@
++++
+title = "Working with Grafana panels"
+aliases = ["docs/grafana/latest/panels/panel-editor/", "docs/grafana/latest/panels/add-a-panel/", "docs/grafana/latest/panels/standard-options/", "docs/grafana/latest/panels/panel-options/", "docs/grafana/latest/panels/legend-options/"]
+weight = 100
++++
+
+# Working with Grafana panels
+
+A Grafana panel is the user interface you use to define a data source query, and transform and format data that appears in visualizations.
```
[the remark this answers is not in the document]
Comment (chri2547) [reply docs/sources/panels/working-with-panels/_index.md:9]: I totally agree, and the key phrase you use is "depending on what they want". I think this is where we add content that expands the notion of queries to include _the business problem the user is trying to solve_. Let's talk about writing content outside of this PR that will tie queries, formatting, overrides, visualization selection back to business problems, and how to plan for dashboards and writing queries. And have lots of rich examples.

File: docs/sources/panels/working-with-panels/_index.md [hunk 10]
```diff
@@ -0,0 +1,15 @@
...
+
+A panel editor includes a query builder and a series of options that you can use to transform data and add information to your panels.
+
+This section includes the following topics:
+
+{{< section >}}
```
[the remark this answers is not in the document]
Comment (chri2547) [reply docs/sources/panels/working-with-panels/_index.md:15]: Let's sync about this one. Add a panel is after 'navigate', which shows them the UI. That said, I'll move navigate the inspector down a bit and will bump add a title and description up.

File: docs/sources/panels/working-with-panels/navigate-panel-editor.md [hunk 11]
```diff
@@ -0,0 +1,34 @@
...
+- **Time range controls -** For more information, refer to [Time range controls]({{< relref "../../dashboards/time-range-controls.md" >}}).
+
+3. Data section: The data section contains tabs where you enter queries, transform your data, and create alert rules (if applicable).
+
+- **Query tab -** Select your data source and enter queries here. For more information, refer to [Add a query]({{< relref "../query-a-data-source/add-a-query.md" >}}).
+
+- **Transform tab -** Apply data transformations. For more information, refer to [Transform data]({{< relref "../transform-data/_index.md" >}}).
+- **Alert tab -** Write alert rules. For more information, refer to [Overview of Grafana 8 alerting]({{< relref "../../alerting/unified-alerting/_index.md" >}}).
+
+4. Panel display options: The display options section contains tabs where you configure almost every aspect of your data visualization. Not all options are available for each visualization.
```
[the remark this answers is not in the document]
Comment (chri2547) [reply docs/sources/panels/working-with-panels/navigate-panel-editor.md:32]: Added a few links.

File: docs/sources/panels/working-with-panels/format-standard-fields.md [hunk 12]
```diff
@@ -0,0 +1,31 @@
++++
+title = "Format a standard field"
+aliases = []
+weight = 40
++++
+
+# Format a standard field
```
[the remark this answers is not in the document]
Comment (chri2547) [reply docs/sources/panels/working-with-panels/format-standard-fields.md:7]: Yes, let's work on that in another PR.

File: docs/sources/panels/working-with-panels/add-link-to-panel.md [hunk 13]
```diff
@@ -0,0 +1,11 @@
++++
+title = "Add a link to a panel"
+aliases = []
+weight = 60
++++
+
+# Add a link to a panel
```
[the remark this answers is not in the document]
Comment (chri2547) [reply docs/sources/panels/working-with-panels/add-link-to-panel.md:7]: Let's not for now. that topic contains a link to an 'add link' topic outside of the panels docs. I haven't ventured to make any changes, thinking we can tackle it after this PR.

**Comment by torkelo** [comment]
Looks like a merged was done inccorrectly, as the PR diff contains chains from unrelated PRs. 

Just wanted to add a friendly reminder to add aliases for any renamed files so that old links to pages still work

**Comment by chri2547** [comment]
> Looks like a merged was done inccorrectly, as the PR diff contains chains from unrelated PRs.
> 
> Just wanted to add a friendly reminder to add aliases for any renamed files so that old links to pages still work

Hi Torkel, my bad - I've actually created a new PR, started from scratch and will be closing this one. As for aliases, because these docs have changed dramatically in structure, I added aliases to the section landing pages as there wasn't a 1:1 match at the file level.

File: docs/sources/panels/working-with-panels/download-log-results.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (chri2547) [reply docs/sources/panels/working-with-panels/download-log-results.md:2]: Change to 'download query results'

**Pull request closed without merging**